cmd/main.go

```diff
@@ -73,11 +74,42 @@ func main() {
 		}
 	})
 
+	//Liveness probe endpoint
+	http.HandleFunc("/liveness", func(w http.ResponseWriter, _ *http.Request) {
```

Comment:
```suggestion
	http.HandleFunc("/healthy", func(w http.ResponseWriter, _ *http.Request) {
```

I would say the endpoint should be `/healthy` to keep consistency with `/ready`. WDYT?

Reply:
renamed, closing